src/Features/Core/Portable/GenerateFromMembers/AbstractGenerateFromMembersCodeRefactoringProvider.cs

```diff
@@ -52,7 +52,7 @@ protected AbstractGenerateFromMembersCodeRefactoringProvider()
             var semanticFacts = document.GetLanguageService<ISemanticFactsService>();
 
             var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
-            var selectedDeclarations = syntaxFacts.GetSelectedMembers(root, textSpan);
+            var selectedDeclarations = syntaxFacts.GetSelectedMembers(root, textSpan, canOverlap: true);
```

Comment:
`canOverlap` seems a little bit confusing as it implies couple of spans, but the API only passes a single span. How about something like `allowPartialSelection`? Regardless, we should add a doc comment on every API with this parameter, and a comment at the place we use the parameter to do overlaps check.

Reply:
Another question: Is it possible to make it a non-optional parameter? Generally, preference is to avoid adding optional parameters to internal/private APIs, unless you have tons of existing callsites that you don't want to manually update to explicitly pass the default.

Reply:
Good idea.  I changed the name and made it non-optional.  There are three features that use this method  - "Add parameter to constructor", "Generate constructor", and "Generate equals and GetHashCode".  I am allowing partial selections for the 2 constructor refactorings, but not for the generate equals refactoring.

Reply:
Sounds good to me, thanks!